Header header
float64 heading_cost
float64 heading_error
float64 velocity_cost
float64 clearance_cost
float64 dwa_loop_time